config.go

```diff
@@ -15,7 +16,10 @@ type Config struct {
 	WalletConfig      *nkn.WalletConfig
 	DialConfig        *nkn.DialConfig
 	TunaSessionConfig *ts.Config
+	UDP               bool
+	UDPIdleTime       int64 // Seconds. Time to purge idle udp connecitons, 0 is for no purge.
```

Comment:
connections

Reply:
Use int32 instead for better gomobile compatability

Reply:
Yes, modified.